server/src/middleware/tests/AdminController.test.ts

```diff
@@ -503,22 +430,14 @@ describe("AdminController endpoint tests", () => {
 
       expect(dates).toHaveLength(2)
       expect(dates[0].max_bookings).toBeLessThanOrEqual(0)
-      expect(dates[0].description).toEqual("my test")
       expect(removeUnderscoresFromTimestamp(dates[0].date)).toEqual(startDate)
 
       expect(dates[1].max_bookings).toBeLessThanOrEqual(0)
-      expect(dates[1].description).toEqual("skipped a date")
```

Comment:
why are these deleted?

Reply:
Oh whoops, I saw an earlier test and thought that it would be working for every case afterwards 😬